src/plugins/data_source_management/public/components/text_content/text_content.ts

```diff
@@ -14,22 +14,25 @@ export const deleteText = i18n.translate('delete', {
   defaultMessage: 'Delete',
 });
 
-export const titleText = i18n.translate('title', {
+export const TITLE = i18n.translate('title', {
   defaultMessage: 'Title',
 });
 
-export const descriptionText = i18n.translate('description', {
+export const DESCRIPTION = i18n.translate('description', {
   defaultMessage: 'Description',
 });
 
-export const usernameText = i18n.translate('username', {
+export const USERNAME = i18n.translate('username', {
   defaultMessage: 'Username',
 });
 
-export const passwordText = i18n.translate('password', {
+export const PASSWORD = i18n.translate('password', {
   defaultMessage: 'Password',
 });
 
+export const OPTIONAL = i18n.translate('optional', {
+  defaultMessage: 'optional',
+});
 /* Datasource listing page */
 export const dsListingAriaRegion = i18n.translate(
```

Comment:
Can we make sure the variable names formating is consistent?

Reply:
I talked with Manny offline about this variable names format and we decide to make variables all capitalized. To avoid affecting each other, I changed part of variable names that I used in create page and Manny will change remain variables. We'll make sure all variable names formatting is consistent.